src/test/java/core/basesyntax/service/operations/BalanceOperationTest.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class BalanceOperationTest {
+    private FruitTransaction transaction;

```

Comment:
The 'transaction' object is not initialized in a setup method. Consider creating a setup method, annotated with @BeforeEach, to initialize this object for each test.

Reply:
ok, done